src/main/resources/public/metrics.html

```diff
@@ -0,0 +1,159 @@
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<!DOCTYPE html>
+<html lang="en">
+    <script src="./js/jquery.min.js"></script>
+    <script src="./js/nna.utility.js"></script>
+    <script type="text/javascript">checkIfAuthenticated();</script>
+    <head>
+        <!-- Bootstrap core CSS -->
+        <link href="./css/bootstrap.min.css" rel="stylesheet">
+        <!-- Sweetalert core CSS -->
+        <link href="./css/sweetalert.css" rel="stylesheet">
+        <!-- Custom styles for this template -->
+        <link href="./css/nna.css" rel="stylesheet">
+        <link href="./css/datatables.min.css" rel="stylesheet">
+    </head>
+    <body>
+        <div class="container">
+            <div id="navbar"></div>
+            <!-- Main component for a primary marketing message or call to action -->
+            <div class="jumbotron">
+                <h1>User Metrics</h1>
+            </div>
+            <div id="userMetrics">
+                <table id="userDataTable" class="table table-bordered" style="width: 100%;">
+                    <thead>
+                    <tr>
+                        <th></th>
+                        <th>Username</th>
+                        <th>IP Address</th>
+                        <th>Login Count</th>
+                        <th>Logout Count</th>
+                        <th>Query Count</th>
+                    </tr>
+                    </thead>
+                </table>
+            </div>
+        </div> <!-- /container -->
+        <!-- Bootstrap core JavaScript
+        ================================================== -->
+        <!-- Placed at the end of the document so the pages load faster -->
+        <script src="./js/bootstrap.min.js"></script>
+        <script src="./js/sweetalert.min.js"></script>
+        <script src="./js/Chart.bundle.min.js"></script>
+        <script src="./js/cycle.js"></script>
+        <script src="./js/datatables.min.js"></script>
+        <script type="text/javascript">
+            $(document).ready(function() {
+                $('#navbar').load('navbar.html', function() {
+                    checkIfAdmin();
+                });
+            });
+        </script>
+        <script type="text/javascript">$(document).ready(function(){  $("head").append("header.html");});</script>
+        <script type="text/javascript">getClusterName();</script>
```

Comment:
In order to utilize this call on your page you need a div with id `connection`. Check `suggestion.html` as an example.